solidity/contracts/bridge/Bridge.sol

```diff
@@ -246,6 +251,11 @@ contract Bridge is Ownable, EcdsaWalletOwner {
     ///         to the redeemer in full amount.
     uint256 public redemptionTimeout;
 
+    /// TODO: Make it governable.
+    /// @notice Maximum amount of the total BTC transaction fee that is
+    ///         acceptable in a single moving funds transaction.
+    uint64 public movingFundsTxMaxTotalFee;
```

Comment:
Nope, there can be multiple outputs. And it is hard to reason about fee per output. This is why I went with the total transaction fee since it makes more sense with moving funds. We do not do any bookkeeping with the received amounts and we just check if the entire transaction has not burned too much amount for fees.

Reply:
Yeah, you are right. I should have my coffee first. Can we make it super-explicit in docs?

I propose adding `@dev` section to `depositTxMaxFee`, `redemptionTxMaxFee`, and `movingFundsTxMaxTotalFee` like:

```
    /// (...)
    /// @dev This is a per-deposit input max fee for the sweep transaction.
    uint64 public depositTxMaxFee;

    /// (...)
    /// @dev This is a per-redemption output max fee for the redemption transaction.
    uint64 public redemptionTxMaxFee;

    /// (...)
    /// @dev This is a TOTAL max fee for the move funds transaction. Note that
    ///      `depositTxMaxFee` is per single deposit and `redemptionTxMaxFee`
    ///      if per single redemption. `movingFundsTxMaxTotalFee` is a total fee
    ///      for the entire transaction.
    uint64 public movingFundsTxMaxTotalFee;
```

Reply:
See https://github.com/keep-network/tbtc-v2/pull/184/commits/8cd1049ee269075e74db2c66626e00216e55a39b